m1well.Utils/plugin.json

```diff
@@ -0,0 +1,39 @@
+{
+  "noteplan.minAppVersion": "3.0.25",
+  "macOS.minVersion": "10.13.0",
+  "iOS.minVersion": "14",
+  "plugin.id": "m1well.Utils",
+  "plugin.name": "🛠 Utils",
+  "plugin.description": "Plugin for some Utils like repeat selected task, etc.",
+  "plugin.author": "@m1well",
+  "plugin.url": "https://github.com/NotePlan/plugins/blob/main/m1well.Utils/README.md",
+  "plugin.version": "1.0.0",
+  "plugin.dependencies": [],
+  "plugin.script": "script.js",
+  "plugin.commands": [
+    {
+      "name": "repeater",
+      "alias": ["well:repeater", "well"],
+      "description": "Repeat task(s) just by selecting them in whatever note you want with a lot of functionalities",
+      "jsFunction": "repeater"
+    },
+    {
+      "name": "sorter",
```

Comment:
to not confuse users, this should be called "sort selected tasks" to be clear that this relates to selections only (and tasks)

Reply:
it is not only for tasks ;)
but ok I change it to `selectionSorter` :)